src/main/psxsdk/libsnd/vmanager.c

```diff
@@ -291,7 +331,39 @@ s16 SpuVmGetSeqRVol(s16 arg0) {
     return new_var[((s32)(arg0 & 0xFF00)) >> 8].unk76;
 }
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/vmanager", SpuVmSeqKeyOff);
+void SpuVmSeqKeyOff(s16 arg0) {
+    u8 voice;
+    s32 bitsUpper;
+    s32 bitsLower;
+    u16 temp2;
+    voice = 0;
+    if (spuVmMaxVoice != 0) {
```

Comment:
This can be converted as a `for`

Reply:
Not getting a match here https://decomp.me/scratch/VSzmz

Reply:
You need to remove the `if`: https://decomp.me/scratch/U2Zir